apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -52,10 +52,10 @@ public class BigtableClientSnippets
                 "user12345",
                 // If the latest value in the "Address:Country" column is "USA"...
                 RowFilters.Chain(
-                    RowFilters.FamilyNameRegex("Address"),
-                    RowFilters.ColumnQualifierRegex("Country"),
+                    RowFilters.FamilyNameExact("Address"),
```

Comment:
Having changed all these uses, do we have tests for actual regex patterns?

Reply:
Yes, there are 7 calls into `...Regex` helpers in Tests and 5 in IntegrationTests.

Reply:
SGTM.